test/resources/microshift-process.resource

```diff
@@ -80,6 +77,16 @@ Restart MicroShift
     [Documentation]    Restart the MicroShift service
     ${ushift_pid}=    MicroShift Process ID
     ${ushift_etcd_pid}=    MicroShift Etcd Process ID
+    # Force delete of core pods to avoid extended downtime after a restart.
+    # Whenever there is apiserver unavailability some of these pods enter a
+    # crash loop because they can not update/get resources they need from the
+    # cluster. This backoff may cap at 5min and trigger greenboot failures.
+    # Manifests must be deleted or they may be lingering pods trying to query
+    # apiserver when its up again, triggering transient errors regarding access
+    # to certain resources. The safest option is to remove them before restart
+    # and start clean.
+    Setup Kubeconfig
+    Delete Core Components
```

Comment:
Are we not creating a race condition by doing this before the restart? I mean, part of those pods may be recreated before the actual shutdown takes place, right?

Reply:
Would moving it after `Wait For MicroShift` have the same effect?

Reply:
The pod deletion is triggered by deleting the top level resource, which is the daemonset/deployment, so they can not be recreated.